doc/api/async_hooks.md

```diff
@@ -29,7 +29,7 @@ Following is a simple overview of the public API.
 const async_hooks = require('async_hooks');
 
 // Return the ID of the current execution context.
-const cid = async_hooks.currentId();
+const cid = async_hooks.currentAsyncId();
```

Comment:
@AndreasMadsen , maybe `async_hooks.currentContext.asyncId` and `async_hooks.currentContext.triggerId` more make sense?  But it will add a new object...